tools/generate_property_translations.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python3
+#
+# Generates translation strings from gecko property files for lupdate
+# to process
+
+import os, os.path, sys, re, dataclasses, functools
+
+PREFIX = "sailfish_components_webview_popups-la"
```

Comment:
This could be an argument then with that this would be easier to utilize for other cases as well.

Reply:
I added proper argument parsing and --prefix|-p for this one. Defaults to the old value.